x/liquidstakeibc/types/msgs.go

```diff
@@ -310,6 +310,24 @@ func (m *MsgUpdateHostChain) ValidateBasic() error {
 			if entries <= 0 {
 				return fmt.Errorf("max entries undelegation/redelegation cannot be zero or lesser, found %v", entries)
 			}
+		case KeyUpperCValueLimit:
+			limit, err := sdk.NewDecFromStr(update.Value)
+			if err != nil {
+				return fmt.Errorf("unable to parse string to sdk.Dec: %w", err)
+			}
+
+			if limit.LT(sdk.OneDec()) {
+				return sdkerrors.ErrInvalidRequest.Wrapf("invalid c_value upper limit value, should be 1 <= limit")
+			}
+		case KeyLowerCValueLimit:
+			limit, err := sdk.NewDecFromStr(update.Value)
+			if err != nil {
+				return fmt.Errorf("unable to parse string to sdk.Dec: %w", err)
+			}
+
+			if limit.GT(sdk.OneDec()) || limit.IsNegative() {
```

Comment:
negative is fine, hard higher cap is fine to 1.1

Reply:
talking about `limit.GT(sdk.OneDec()) `

Reply:
it should never be higher than one, will start at smth like 0.99 all the way to 0, that is why the check

Reply:
heavy slashing on multiple validators very first day or something..

Reply:
but lower is still ok, i was thinking more of c_value going to 1.02, then lower would be 1.01, higher 1.03, we want to avoid any jumps to the value, can only be gradual

Reply:
in that case, since we can't compare with the state in that check , let's just not check for range (apart from negative) and just for valid number

Reply:
yes, add that in msg_server, that higher should be greater than lower. and vice versa